sale_order_variant_mgmt/models/sale_order.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 Pedro M. Baeza <pedro.baeza@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from openerp import models, fields
+
+
+class SaleOrderLine(models.Model):
+    _inherit = 'sale.order.line'
+
+    # This field is for avoiding conflicts with sale_stock module, that
+    # adds product_tmpl_id, and its possible modifications. This field name
+    # for sure won't conflict
+    product_tmpl_id_sale_order_variant_mgmt = fields.Many2one(
```

Comment:
It's a long history: this field is used in product configurator modules that have a common abstract model using product_tmpl_id field as a regular field. sale_stock_screws up this, so I need to fix it through the glue module https://github.com/odoomrp/odoomrp-wip/tree/8.0/sale_stock_product_variants. I don't want to need another glue module, and as this is only a wildcard field, it doesn't matter the name.